handbook/people-ops/compensation.md

```diff
@@ -1,10 +1,16 @@
 # Compensation
 
-## Salary
+Our compensation plan includes three components: cash (salary and commissions), equity, and benefits/perks.
 
-We pay competitive salaries because we want teammates focused on helping Sourcegraph succeed, not worrying about paying their bills.
+We use market data from various sources (e.g. venture capital firms, 3rd-party consultants) to determine the market rate for each role and level within that role.
+
+To provide competitive and generous compensation (both cash and equity) to everyone on our [all remote team](../../company/remote/index.md) (regardless of location), our compensation is benchmarked against market rates in San Francisco, California ([one of the most expensive cities in the world](https://www.businessinsider.com/how-expensive-is-san-francisco-mind-blowing-facts-2019-5)).
+
+## Cash (salary and commission)
+
+We pay competatively because we want to hire the best people and we want our teammates to be focused on helping Sourcegraph succeed, not worrying about paying bills.
```

Comment:
```suggestion
We pay competitively because we want to hire the best people and we want our teammates to be focused on helping Sourcegraph succeed, not worrying about paying bills.
```